site/docs/latest/clients/go.md

```diff
@@ -0,0 +1,589 @@
+---
+title: The Pulsar Go client
+tags: [client, go, golang]
+---
+
+<!--
+
+    Licensed to the Apache Software Foundation (ASF) under one
+    or more contributor license agreements.  See the NOTICE file
+    distributed with this work for additional information
+    regarding copyright ownership.  The ASF licenses this file
+    to you under the Apache License, Version 2.0 (the
+    "License"); you may not use this file except in compliance
+    with the License.  You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing,
+    software distributed under the License is distributed on an
+    "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+    KIND, either express or implied.  See the License for the
+    specific language governing permissions and limitations
+    under the License.
+
+-->
+
+The Pulsar Go client can be used to create Pulsar [producers](#producers), [consumers](#consumers), and [readers](#readers) in Go (aka Golang).
+
+{% include admonition.html type="info" title="API docs available as well"
+   content="For standard API docs, consult the [Godoc](https://godoc.org/github.com/apache/incubator-pulsar/pulsar-client-go/pulsar)." %}
+
+## Installation
+
+You can install the `pulsar` library locally using `go get`:
+
+```bash
+$ go get -u github.com/apache/incubator-pulsar/pulsar-client-go/pulsar
```

Comment:
I got following errors. 
```
# go get -u github.com/apache/incubator-pulsar/pulsar-client-go/pulsar
# github.com/apache/incubator-pulsar/pulsar-client-go/pulsar
In file included from src/github.com/apache/incubator-pulsar/pulsar-client-go/pulsar/c_client.go:24:0:
./c_go_pulsar.h:22:29: fatal error: pulsar/c/client.h: No such file or directory
 #include <pulsar/c/client.h>
                             ^
compilation terminated.
```

Do I need additional settings?

Go version:  
go1.10 linux/amd64
go1.10.2 linux/amd64

Reply:
I’m having the same issue when trying to install via go get

Reply:
I included `-I${GOPATH}/src/github.com/apache/incubator-pulsar/pulsar-client-cpp/include` in `CGO_CFLAGS` like https://github.com/apache/incubator-pulsar/pull/1764/commits/d1b2ea3e04a6c4d55f935c3aef1aac831aceab8d , then this issue didn't occur.